### PR TITLE
libxo: update to 1.6.0; install .a archive for -devel

### PR DESCRIPTION
**Pull request by arp242**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by mhmdanas** [comment]
Is there a reason you want the static libraries?

**Comment by arp242** [comment]
I want to statically link things against libxo.

**Comment by ahesford** [comment]
Adding static libs is fine. Unless there was some specific reason this was disabled to begin with, it the "disable static" option shouldn't have been added.

File: srcpkgs/libxo/template [hunk 1]
```diff
@@ -32,6 +31,7 @@ libxo-devel_package() {
 		vmove usr/lib/pkgconfig
 		vmove usr/share/man/man3
 		vmove "usr/lib/*.so"
+		vmove "usr/lib/*.a"
 		vmove usr/share/libxo
 		vmove "usr/lib/libxo/encoder/*.so"
```
Comment (classabbyamp) [inline srcpkgs/libxo/template:36]:
```suggestion
		vmove "usr/lib/libxo/encoder/*.so"
		vmove "usr/lib/libxo/encoder/*.a"
```

**Pull request closed without merging**